### PR TITLE
add proxy monitor config

### PR DESCRIPTION
**Pull request by mixedpuppy**
Other config files are here https://github.com/mozilla-extensions/proxy-monitor/pull/6

### CODE REVIEW

**Review by escapewindow — CHANGES_REQUESTED** [review]
It looks like we need to add `proxymonitor` to taskcluster/ci/config.yml.

**Comment by mixedpuppy** [comment]
> It looks like we need to add `proxymonitor` to taskcluster/ci/config.yml.

bah, forgot to git add that file.

File: taskcluster/ci/config.yml [hunk 1]
```diff
@@ -117,6 +117,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/news-disinformation-study
             default-ref: master
             type: git
+        proxymonitor:
+            name: "Proxy Monitor extension"
+            project-regex: proxymonitor$
+            default-repository: https://github.com/mozilla-extensions/proxy-monitor
```
Comment (escapewindow) [inline taskcluster/ci/config.yml:123]: This looks like a private repo, so we'll need `git@` URL formatting or the task will hang trying to clone.

**Review by escapewindow — CHANGES_REQUESTED** [review]

File: taskcluster/ci/config.yml (hunk 1 above)
Comment (escapewindow) [inline taskcluster/ci/config.yml:123]:
```suggestion
            default-repository: git@github.com:mozilla-extensions/proxy-monitor
```

File: manifests/proxy-monitor.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description: Direct Failover for system requests.
+repo-prefix: proxymonitor
+active: true
+private-repo: true
+branch: main
+artifacts:
+  - web-ext-artifacts/proxy-monitor.xpi
+addon-type: system
+install-type: npm
```
Comment (escapewindow) [inline manifests/proxy-monitor.yml:9]:
```suggestion
install-type: npm
docker-image: node-lts-latest
```

**Review by escapewindow — APPROVED** [review]
Just needed 2 lines of changes, plus a merge of master.

**Pull request merged**